package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-treebeard",
-  "version": "2.0.2",
+  "version": "2.0.3",
```

Comment:
Please don't increment the version number on every PR.
I'd like to do it in it's own commit.

That way we can squash multiple PR's/commits into the same release